scripts/scil_surface_apply_transform.py

```diff
@@ -68,23 +71,23 @@ def main():
     args = parser.parse_args()
     logging.getLogger().setLevel(logging.getLevelName(args.verbose))
 
-    assert_inputs_exist(parser, [args.in_surface, args.ants_affine],
-                        args.ants_warp)
+    assert_inputs_exist(parser, [args.in_moving_surface, args.in_transfo],
+                        args.in_deformation)
     assert_outputs_exist(parser, args, args.out_surface)
 
     # Load mesh
-    mesh = load_mesh_from_file(args.in_surface)
+    mesh = load_mesh_from_file(args.in_moving_surface)
```

Comment:
A flip xy must be done after loading to go to RAS (from LPS) and the same flip must be applied before saving to go from RAS to LPS